projects/views.py

```diff
@@ -8,10 +10,17 @@
 
 class ProjectCreate(CreateView):
     model = Project
-    fields = ['title', 'description', 'max_members', 'difficulty',
-              'image', 'participants', 'coach', 'owner']
+    fields = ['title', 'description', 'max_members', 'difficulty', 'image']
     success_url = reverse_lazy('project_list')
 
+    @method_decorator(login_required)
```

Comment:
what are these method doing?

Reply:
making sure all the post/get methods will be require login first, otherwise it will fail with 401 I think